tutorindigo/templates/indigo/lms/static/sass/courseware/_discover.scss

```diff
@@ -136,6 +139,104 @@
         clear: both;
         content: "";
     }
+    .content-area {
+        display: flex; // Align the sidebar and courses horizontally
+        flex-direction: row;
+        gap: 3vw; // Add spacing between the sidebar and courses
+        align-items: flex-start;
+
+        .course-holder {
+            flex-grow: 1; // Allow the courses to take up the remaining space
+            width: 100%; // Ensure it spans the available space
+        }
+    }
+    .filter-sidebar {
+        flex: 0 0 25%; // Set the sidebar to always occupy 25% of the width
+        max-width: 25%;
```

Comment:
```suggestion
        max-width: 100%;
```


This change is not mandatory but I think looks better

Reply:
I couldn't make it look good. I also use your responsive suggestion, but it doesn't work. I obtain:
![image](https://github.com/user-attachments/assets/7cde5b02-b02a-41e9-91d6-96344b85e3ec)

Reply:
Without the changes, but using part of your responsive suggestion 🙌
![image](https://github.com/user-attachments/assets/4b862742-c57e-479e-8cc8-c8f3905b3956)
![image](https://github.com/user-attachments/assets/49309530-ab62-434b-9cfb-5b4fe6811683)

Reply:
I'll resolve the comment and merge!
For further improvements, we can use another PR!

Thanks again for everything.